spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/core/AbstractMqttClientManager.java

```diff
@@ -53,6 +53,10 @@
 
 	private int phase = DEFAULT_MANAGER_PHASE;
 
+	private long completionTimeout = ClientManager.DEFAULT_COMPLETION_TIMEOUT;
```

Comment:
Consider using the more modern `Duration`?

Reply:
I will, when the target client lib does, but now for consistency I would stay with the same type the target API expects from us.